doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -39,6 +39,7 @@ IDE, and tool support
 Board support
 =============
 
+* Added TF-M support to the :ref:`nRF54L15 DK <zephyr:nrf54l15dk_nrf54l15>` (board target ``nrf54l15dk/nrf54l15/cpuapp/ns``), replacing the nRF54L15 PDK (board target ``nrf54l15pdk/nrf54l15/cpuapp/ns``).
```

Comment:
Not sure if this is the correct place... Would it make more sense to have it under Security section?

Replacing the Moonlight PDK with DK on samples might be a good idea to mention as well, under the respective samples sections.

Reply:
So it would instead be listed on every affected sample category? (for now TF-M, crypto and Matter)

Reply:
That was the thought.
By the way, it looks like this is already done for Matter: https://github.com/nrfconnect/sdk-nrf/pull/17298/files (see release-notes-changelog.rst)

Reply:
Did that.